mod/wikirate/set/right/my_contributions.rb

```diff
@@ -22,15 +27,22 @@ def sections
 
 format :html do
   view :core do |args|
-    card.sections.sort.reverse.map do |count, name, contr_name|
-      section_args = {:view=>:open, :title=>name, :hide=>'menu'}
-      # FIXME - cardname
-      if name == 'Initiative'
-        nest Card.fetch(contr_name), section_args.merge(:item=>{:view=>:content, :structure=>'initiative item'})
-      else
-        nest Card.fetch(contr_name), section_args
-      end
-    end.join "\n"
+    if card.sections
+      card.sections.sort.reverse.map do |count, name, contr_name|
+        section_args = { view: :open, title: name, hide: 'menu' }
+        # FIXME - cardname
+        if name == 'Initiative'
+          nest Card.fetch(contr_name), section_args.merge(item: {
+              view: :content, structure: 'initiative item'
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.